src/test/java/org/wso2/carbon/connector/integration/test/AmazonLambdaAuthConnectorIntegrationTest.java

```diff
@@ -0,0 +1,751 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.connector.integration.test;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.connector.integration.test.base.ConnectorIntegrationTestBase;
+import org.wso2.connector.integration.test.base.RestResponse;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AmazonLambdaAuthConnectorIntegrationTest extends ConnectorIntegrationTestBase {
+
+    private Map<String, String> eiRequestHeadersMap = new HashMap<String, String>();
+
+    /**
+     * Set up the environment.
+     * @throws Exception
+     */
+    @BeforeClass(alwaysRun = true)
+    public void setEnvironment() throws Exception {
+
+        String connectorName = System.getProperty("connector_name") + "-connector-" +
+                System.getProperty("connector_version") + ".zip";
+        init(connectorName);
+        getApiConfigProperties();
+
+
+        //eiRequestHeadersMap.put("Accept-Charset", "UTF-8");
+        eiRequestHeadersMap.put("Content-Type", "application/json");
+    }
+
+    /**
+     * Positive test case for createFunction method with mandatory parameters.
+     *
+     */
+    @Test(groups = {"wso2.ei"}, description = "amazonlambda {createFunction} integration test mandatory case.")
+    public void testCreateFunctionWithMandatoryParameters() throws Exception {
+        eiRequestHeadersMap.put("Action", "urn:createFunction");
+        RestResponse<JSONObject> eiRestResponse =
+                sendJsonRestRequest(proxyUrl, "POST", eiRequestHeadersMap,
+                        "createFunction_mandatory.json");
+//        String revisionId = eiRestResponse.getBody().getString("RevisionId");
+//        connectorProperties.setProperty("revisionId", revisionId);
+
```

Comment:
Remove this commented lines if it is not necessary

Reply:
Fix with d580978